php/utils.php

```diff
@@ -100,8 +100,8 @@ function isPathAbsolute($path) {
  * @return [boolean] $is_email True if $email is in email address format
  */
 function isValidEmail($email) {
-  $is_email = filter_var($email, FILTER_VALIDATE_EMAIL);
-  return (boolean)$is_email;
+  $is_email = !is_bool(filter_var($email, FILTER_VALIDATE_EMAIL));
```

Comment:
Oh. oh god. Yep, you're totally right.

Would you mind throwing a unit test around it? https://github.com/pantheon-systems/cli/blob/master/tests/unit_tests/test-utils.php